Neos.Media.Browser/Resources/Public/JavaScript/collections-and-tagging.js

```diff
@@ -1,149 +1,142 @@
-(function ($) {
-    $('.draggable-asset').each(function () {
-        $(this).draggable({
-            helper: 'clone',
-            opacity: 0.3,
-            revert: 'invalid',
-            revertDuration: 200,
-            start: function (event, ui) {
-                $(event.toElement).closest('a').one('click', function (e) {
-                    e.preventDefault();
-                });
-            },
-            stop: function (event, ui) {
-                // event.toElement is the element that was responsible
-                // for triggering this event. The handle, in case of a draggable.
-                $(event.toElement).closest('a').off('click');
-            }
-        });
-    });
+window.addEventListener('DOMContentLoaded', (event) => {
+	jQuery(function() {
```

Comment:
The dollar needs to be set as argument:

```suggestion
	jQuery(function($) {
```